### DISCUSSION TITLE
Lab 02.04 - Card Trick

### DISCUSSION THREAD

**Discussion by maguiargil** [rramosp/ai4eng.v1#219] [02 - PYTHON]
Cordial saludo,

En el enunciado del laboratorio 02.04 correspondiente a "Card Trick" hacen referencia a un video a mirar para entender la actividad. No encuentro el enlace a dicho video ¿Podrían indicarme por favor si el enlace se encuentra en otro sitio del portal o si el enlace no se encuentra? Muchas gracias.

**Comment by Felipe-RA** [comment]
Saludos @maguiargil 

Todos los vídeos correspondientes a un módulo los encuentras al darle click al la raíz del módulo respectivo.

<img width="691" alt="image" src="https://user-images.githubusercontent.com/49454068/218286451-39970d47-8239-45d8-99a2-23588550b8f0.png">


¡Hasta pronto!
